pkg/compiler/types.go

```diff
@@ -830,6 +830,28 @@ var typeFmtFormat = &typeArg{
 	Kind:  kindIdent,
 }
 
+// typeCompressed is used for compressed data.
+var typeCompressed = &typeDesc{
+	Names:     []string{"compressed"},
+	CantBeOpt: true,
```

Comment:
This allows `compressed[opt]`. Do we want images to be optional?

Reply:
I thought it was the other way around? I.e. `CantBeOpt = true` meants it *can't* be optional (see [here](https://github.com/google/syzkaller/blob/5c716ff64acaf8672a9fe96c21951be48b2b8d5e/pkg/compiler/check.go#L930-L935), and from local testing). Have I missed something?

Reply:
Right. I confused "Cant" with "Can".